charts/valtimo-backend/templates/pvc.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ include "valtimo-backend.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  {{- with .Values.persistence.annotations  }}
+annotations:
+  {{ toYaml . | indent 4 }}
+  {{- end }}
+labels:
+  app: {{ template "valtimo-backend.fullname" . }}
+  chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  release: "{{ .Release.Name }}"
+  heritage: "{{ .Release.Service }}"
+spec:
+  accessModes:
+    - {{ .Values.persistence.accessMode | quote }}
```

Comment:
Will the application be able to handle access-modes other than ReadWriteMany? If not, then it's best to hardcode `ReadWriteMany` here and remove it from the values.yaml. Otherwise there's a risk that the user configures it as readWriteOnce and the application misbehaves as a result.